packages/manager/apps/web/client/app/private-database/order/clouddb/private-database-order-clouddb.html

```diff
@@ -0,0 +1,139 @@
+<oui-page-header
+    data-heading="{{:: 'private_database_order_clouddb_title' | translate }}"
+    data-description="{{:: 'private_database_order_clouddb_subtitle' | translate }}">
+</oui-page-header>
+
+<div class="container-fluid my-4 mx-5">
+    <div data-ovh-alert="cloudDB_order_alert"></div>
+
+    <oui-stepper
+        data-current-index="$ctrl.currentIndex"
+        data-on-finish="$ctrl.validateCheckout()">
+        <oui-step-form
+            data-header="{{:: 'private_database_order_clouddb_database_type' | translate }}"
+            data-on-focus="$ctrl.selectTypeOption()"
+            data-editable="false"
+            data-navigation="false"
+            data-disabled="$ctrl.hasValidatedCheckout"
+            data-valid="$ctrl.model.type">
+            <oui-field data-label="{{:: 'private_database_order_clouddb_database_question' | translate }}">
+                <oui-radio
```

Comment:
There's only one possible choice ?

Reply:
Yes, the choice is auto selected.

Reply:
Then why is it a radio button ? For consistency ?